apps/users/filters.py

```diff
@@ -51,6 +53,7 @@ def participation_filter(self, queryset, name, value):
         choices=(
             ('creator', _('Submitted')),
             ('collaborator', _('Supported')),
+            ('watcher', _('Watched'))
```

Comment:
This should be "Watching". And it should also be "Supporting", because you keep doing that. Well, on second thought, "Supporting" is a very special case, but it should be "Watching" anyway.

Reply:
You right, that is also what the designs say.